objects.go

```diff
@@ -104,6 +104,25 @@ func NewNetwork(nw Network) *Network {
 	return &res
 }
 
+type IPv6Network struct {
```

Comment:
Need to consider using existing object 'Network' for both v4 and v6. Most probably, this will require some discussion.